CorsixTH/Lua/dialogs/build_room.lua

```diff
@@ -96,22 +96,27 @@ function UIBuildRoom:UIBuildRoom(ui)
     _S.room_classes.clinics,
     _S.room_classes.facilities
   }
-  self.category_rooms = {
-  }
+
+  self:updateList()
+
+  self:makeTooltip(_S.tooltip.build_room_window.cost, 160, 228, 282, 242)
+end
+
+function UIBuildRoom:updateList()
```

Comment:
Being a little picky here, as the function name came from my branch - I did say I didn't like it. Technically setCategory updates a member, 'list' from the category_rooms from that this reprocesses. Maybe updateBuildableRooms or something.